TexasHoldem.c

```diff
@@ -398,7 +398,7 @@ void randomCommunityCard(int draw, int communitycards[][2])
 	}
 }
 
-//this function checks to make sure the cards/card chosen by randomCard is not already chosen
+//this function checks to make sure the cards/card chosen by randomCommunityCard is not already chosen
 int CheckRandCommunity(int draw, int communitycards[][2])
```

Comment:
need to put player cards as an input to the function, and change the function prototype and call as well.